mcbackend/adapters/hopsy.py

```diff
@@ -0,0 +1,76 @@
+import numpy
+import numpy as np
+
+import hopsy
+from typing import Dict, List, Optional, Union
+import hagelkorn
+
+from mcbackend.meta import Coordinate, DataVariable, Variable
+
+from ..core import Backend, Chain, Run, RunMeta
```

Comment:
I'll approve the pipeline trigger, but this is going to give you a red ❌ pre-commit via isort:
* Start with standard library imports
* Followed by from standard library imports
* Blank line
* Same for 3rd party
* Blank line
* Local package imports